stripe/src/main/java/com/stripe/android/paymentsheet/PaymentSheetActivity.kt

```diff
@@ -265,6 +265,11 @@ internal class PaymentSheetActivity : BaseSheetActivity<PaymentResult>() {
                 }
             }
         }
+
+        // When using commit on the fragments, the fragment transaction happens
```

Comment:
Is there advantage to conditionally using `commitNow` for API >= 24? I imagine most users will be >= 24.

Reply:
commitNow has the advantage that it will only commit those things in the runnable.  executePendingTransactions will execute all the pending fragments, not just the one you just commited.

Reply:
I'll leave it to your discretion as to whether we want to do this or not.